platform/install/environments/gcp.mdx

```diff
@@ -0,0 +1,292 @@
+import ListHelmVersions from '../../_partials/install/list-helm-versions.mdx';
+import BasePrerequisites from '../../_partials/install/base-prerequisites.mdx';
+import InstallCli from '../../../vcluster/_partials/deploy/install-cli.mdx';
+import KubeconfigUpdate from '../../../docs/_partials/kubeconfig_update.mdx';
+import ManagedK8sInstallVcluster from '../../../docs/_partials/managed_k8s_install_vcluster.mdx';
+import ProAdmonition from '../../../vcluster/_partials/admonitions/pro-admonition.mdx';
+
+import Flow, { Step } from '@site/src/components/Flow';
+
+import Label from '@site/src/components/Label';
+
+
+This guide provides step-by-step instructions for deploying `vCluster platform` on [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine).
+
+## Prerequisites
+
+Before starting, ensure you have the following tools installed:
+<BasePrerequisites />
+- vCluster CLI <InstallCli />
+- [Google Cloud SDK (`gcloud` CLI)](https://cloud.google.com/sdk/docs/install)
+  :::note
+  Ensure you have the necessary IAM permissions to create clusters and manage cloud services.
+  :::
+
+## Create GKE cluster
+
+<Flow id="create-gke-cluster">
+
+<Step>
+Prepare the environment.
+
+Start by creating a zonal GKE cluster using the `gcloud` CLI. First, set up your environment variables:
+
+:::tip
+Project ID can be found in the Google Cloud Console under the project name.
+Alternatively use `gcloud project list` to list all projects and their IDs.
+To check which project is active, use `gcloud config get-value project`.
+:::
+
+```bash title="Set environment variables"
+export PROJECT_ID=development
+export CLUSTER_NAME=vcluster-demo
+export ZONE=europe-west1-b
+export MACHINE_TYPE=e2-standard-4
+```
+
+Configure `gcloud` and enable the required APIs and set default project:
+
+```bash title="Configure gcloud"
+gcloud config set project $PROJECT_ID
+gcloud services enable container.googleapis.com
+```
+</Step>
+
+<Step>
+Create the cluster.
+
+```bash title="Create GKE cluster"
+gcloud container clusters create $CLUSTER_NAME \
+    --zone $ZONE \
+    --machine-type $MACHINE_TYPE \
+    --num-nodes 2
+```
+
+  :::info
+  This process typically takes about 10-15 minutes.
+  :::
+
+This command creates a GKE cluster named vcluster-demo in the europe-west1-b
+zone with two nodes of type e2-standard-4.
+
+<KubeconfigUpdate />
+
+</Step>
+
+<Step>
+Verify the cluster creation.
+
+Verify that the GKE was created successfully by listing the nodes:
+
+```bash title="List cluster nodes"
+kubectl get nodes
+```
+
+You should see output similar to:
+```
+NAME           LOCATION        MASTER_VERSION      MASTER_IP      MACHINE_TYPE   NODE_VERSION        NUM_NODES  STATUS
+vcluster-demo  europe-west1-b  1.30.5-gke.1443001  35.187.66.218  e2-standard-4  1.30.5-gke.1443001  2          RUNNING
+```
+<ManagedK8sInstallVcluster />
+
+</Step>
+
+</Flow>
+
+## Platform setup
+
+With the GKE cluster up and running, you can now deploy the platform in a few
```

Comment:
Are we trying to set up the platform in the GKE cluster or in the vcluster that you've created in GKE?